deployment/ansible/roles/raster-foundry.app/templates/upstart-app.conf.j2

```diff
@@ -1,5 +1,11 @@
 description "rf-django"
 
+echo "UPSTARTING"
```

Comment:
I don't think `echo` is going to be valid in an Upstart service definition outside of a `script` block. If you want to see `UPSTARTING` in `/var/log/upstart/rf-app.log`, you'll need to put it inside `pre-start script`.